etc/parameters/smd/default.json

```diff
@@ -1,4 +1,32 @@
 {
+  "field_color_detection": {
+    "vision_bottom": {
+      "hue": {
+        "end": 168,
+        "start": 43
+      },
+      "luminance": {
+        "end": 225,
+        "start": 23
+      },
+      "saturation": {
+        "start": 65
+      }
+    },
+    "vision_top": {
+      "hue": {
+        "end": 232,
+        "start": 50
+      },
+      "luminance": {
+        "end": 190,
+        "start": 43
```

Comment:
This seems odd 🤔 Why not up to 255?

Reply:
Changed that

Reply:
Hey, that was just a question! 😂 But okay, fine